### PR TITLE
refactor: add packages to @veramo scope

### PR DESCRIPTION
**Pull request by mirceanis**
This is a big refactoring of the daf project to move the packages to the `@veramo` scope.

This should be viewed as an initial commit to the `veramo` project.

Packages were renamed to better reflect their usage and/or purpose and to group them by functionality.

| old | new |
| --- | --- |
| `daf-core`                     | `@veramo/core` |
| `daf-w3c`                      | `@veramo/credential-w3c` |
| `daf-typeorm`                  | `@veramo/data-store` |
| `daf-did-comm`                 | `@veramo/did-comm` |
| `daf-did-jwt`                  | `@veramo/did-jwt` |
| `daf-identity-manager`         | `@veramo/did-manager` |
| `daf-ethr-did`                 | `@veramo/did-provider-ethr` |
| `daf-web-did`                  | `@veramo/did-provider-web` |
| `daf-resolver`                 | `@veramo/did-resolver` |
| `daf-key-manager`              | `@veramo/key-manager` |
| `daf-libsodium`                | `@veramo/kms-local` |
| `daf-react-native-libsodium`   | `@veramo/kms-local-react-native` |
| `daf-message-handler`          | `@veramo/message-handler` |
| `daf-rest`                     | `@veramo/remote-client` |
| `daf-express`                  | `@veramo/remote-server` |
| `daf-selective-disclosure`     | `@veramo/selective-disclosure` |
| `daf-url`                      | `@veramo/url-handle` |

### CODE REVIEW

File: jest.json [hunk 1]
```diff
@@ -4,8 +4,8 @@
   "collectCoverageFrom": [
     "packages/**/src/**/*.ts",
     "!**/examples/**",
```
Comment (rado0x54) [inline jest.json:6]: The examples folder is also gone, correct?

Comment (mirceanis) [reply jest.json:6]: Correct.

We'll also rename a bunch of example projects and point to them, but that will be done later.

File: packages/cli/default/client.yml [hunk 2]
```diff
@@ -17,11 +17,11 @@ server:
   #   - resolveDid
   #   - dataStoreORMGetIdentifiers
 agent:
-  $require: daf-core#Agent
+  $require: @veramo/core#Agent
```
Comment (simonas-notcat) [inline packages/cli/default/client.yml:20]:
```suggestion
  $require: '@veramo/core#Agent'
```

File: packages/cli/default/client.yml [hunk 3]
```diff
@@ -17,11 +17,11 @@ server:
...
   $args:
     - schemaValidation: false
       plugins:
-        - $require: daf-rest#AgentRestClient
+        - $require: @veramo/remote-client#AgentRestClient
```
Comment (simonas-notcat) [inline packages/cli/default/client.yml:24]:
```suggestion
        - $require: '@veramo/remote-client#AgentRestClient'
```

**Review by simonas-notcat — CHANGES_REQUESTED** [review]

**Review by simonas-notcat — APPROVED** [review]
LGTM

**Pull request merged**